integration-tests/src/main/java/io/druid/testing/clients/QueryResourceTestClient.java

```diff
@@ -93,6 +100,7 @@ private String getBrokerURL()
       );
     }
     catch (Exception e) {
+      log.error("exception: " + e.getMessage());
```

Comment:
we can keep it here for this special reason at error level itself, but in general wouldn't log exceptions when propagating them.
pls change to `log.error(e, "message")` however.